test/python/evaluation/testing_config.py

```diff
@@ -0,0 +1,16 @@
+from src.python import MAIN_FOLDER
+from src.python.evaluation.common.util import EvaluationArgument
+from src.python.review.reviewers.perform_review import OutputFormat
+
+
+def get_testing_arguments(n_args=2) -> dict:
```

Comment:
Oh sorry, please don't use such strange constants. You can use a flag: to_add_ traceback or tool_path, which can be None. Please, rewrite this function